### DISCUSSION TITLE
Timing for individual tokens/words when using NeMo tts/ASR

### DISCUSSION THREAD

**Discussion by Herajy** [NVIDIA-NeMo/NeMo#2786] [Q&A]
Hi All,

When using NeMo to do tts and/or ASR, is it possible to get the timing of each individual token/words? Is there an API for that?

**Comment by titu1994** [comment]
No we don't support that yet

**Reply by Herajy** [reply]
Ok, thanks you. I have found this colab useful in this context:
https://colab.research.google.com/github/NVIDIA/NeMo/blob/main/tutorials/asr/Offline_ASR.ipynb#scrollTo=a-LSg9dSL_O1
Nothing similar in the context of tts?

**Reply by titu1994** [reply]
That does not apply to TTS models. You can try to implement it yourself but there's no general solution.
